app.js

```diff
@@ -1023,7 +1026,7 @@ define(function(require) {
 									accountId: accountId
 								},
 								success: _.flow(
-									_.partial(_.get, _, 'data'),
```

Comment:
`lodash#partial()` is no longer necessary here. Only `mergeResults` would be OK.